### ISSUE TITLE
Provide support to upload multiple customer own certificate

### ISSUE THREAD

**Issue by sherintky** [terraform-google-modules/terraform-google-lb-http#357]
### TL;DR

Unable to attach multiple customer own public ssl certificate with application (classic) loadblancer

### Terraform Resources

```markdown
# Terrafom resource block
resource "google_compute_ssl_certificate" "default" {
  project     = var.project
  count       = var.ssl && length(var.managed_ssl_certificate_domains) == 0 && !var.use_ssl_certificates ? 1 : 0
  name_prefix = "${var.name}-certificate-"
  private_key = var.private_key
  certificate = var.certificate

  lifecycle {
    create_before_destroy = true
  }
}

#variables

variable "private_key" {
  description = "Content of the private SSL key. Required if `ssl` is `true` and `ssl_certificates` is empty."
  type        = string
  default     = null
}

variable "certificate" {
  description = "Content of the SSL certificate. Required if `ssl` is `true` and `ssl_certificates` is empty."
  type        = string
  default     = null
}
```

### Detailed design

```markdown
Please make "private_key" and "certificate" type as list(string) instead of string
```

### Additional information

We are unable to upload multiple third party (custom own) public ssl certificate as parameter "certificate" and "private_key" are string type, hence we are able to attach only single ssl certificate with load balancer. However module support multiple Google-managed ssl certificate using ssl_certificates parameter. Please provide an option to support multiple third party (custom own) public ssl certificate

**Comment by imrannayer** [comment]
@sherintky you can create multiple certificates using `google_compute_ssl_certificate` outside of this module and pass them all as list in `ssl_certificates`
